nvm.sh

```diff
@@ -3350,6 +3358,15 @@ nvm() {
         FLAVOR="$(nvm_node_prefix)"
       fi
 
+      # Write .nvmrc if flag is set
+      if [ $NVM_WRITE_TO_NVMRC -eq 1 ] && [ "${VERSION}" != '∞' ] && [ "${VERSION}" != 'N/A' ]; then
+        echo "${VERSION}" | tee "${PWD}"/.nvmrc > /dev/null || {
+          nvm_err "Warning: Unable to write version number (${VERSION}) to .nvmrc"
+          exit 3
+        }
+        nvm_echo "Wrote version number (${VERSION}) to .nvmrc"
+      fi
```

Comment:
this should only happen if the version is already installed, or is successfully installed - not if installation fails.